circle.yml

```diff
@@ -138,6 +138,18 @@ commands:
           working_directory: ~/build
           command: ctest -E 'unittests|smoketests' -j4 --schedule-random --output-on-failure
 
+  collect_coverage_gcc:
```

Comment:
This is refactoring. This step runs twice now. It additionally also rests the coverage counters and the HTML report so the next test suite executed is counted from 0.